02-express-tutorial/app.js

```diff
@@ -1 +1,60 @@
-console.log('Express Tutorial')
+const express = require('express');
+
+const app = express();
+
+app.use(express.static("./public"));
+
+app.get('/api/v1/test', (req, res) => {
+    res.json({ message: "it worked!"});
+});
+
+//Route to get a product by ID
+app.get('/api/v1/products/:productID', (req, res) => {
+    const { productID } = req.params;
+    const idToFind =parseInt(req.params.productID, 10);
+   
+    const product = products.find((p) => p.id === idToFind);
+
+    if (!product) {
+        return res.status(404).json({ error: `Product with ID ${idToFind}`})
+    }
+
+    res.json(product);
+});
+
+app.get('/api/v1/query', (req, res) => {
+    const { search, limit, priceLessThan } = req.query;
+    let filteredProducts = [...products];
+
+    if (search) {
+        filteredProducts = filteredProducts.filter((product) =>
+            product.name.toLowerCase().includes(search.toLowerCase())  
+    );
+    }
+
+    if (pricelessThan) {
```

Comment:
`pricelessThan` should be `priceLessThan` here since that's how it's defined on line 26.

Reply:
I have made the correction